### ISSUE TITLE
Adding reply comment 

### ISSUE THREAD

**Issue by ogahozy** [miguelgrinberg/flasky#357]
Sir, it is possible to add reply under the comment as seen sololearn android app?

**Comment by miguelgrinberg** [comment]
Yes. I wrote a blog article on this topic: https://blog.miguelgrinberg.com/post/implementing-user-comments-with-sqlalchemy.

**Comment by ogahozy** [comment]
Thank you guru. I will close the issue and ask other questions if any at that post page
